src/snoowrap.js

```diff
@@ -268,6 +268,8 @@ const snoowrap = class snoowrap {
   * for "Installed" grant type, needs to be between 20-30 characters long.
   * @param {string} [options.grantType=snoowrap.grantType.INSTALLED_CLIENT] The type of "user-less"
   * token to use {@link snoowrap.grantType}
+  * @param {boolean} options.permanent=true If `true`, the app will have indefinite access. If `false`,
```

Comment:
JSDoc and TS defs are different. TS defs say it's optional, JSDoc claims it's required